src/azure-cli/azure/cli/command_modules/appconfig/tests/latest/export_features_naming.json

```diff
@@ -0,0 +1,26 @@
+{
```

Comment:
If this file is gerated by https://github.com/Azure/azure-cli/blob/2d84719fe003c7fae9a58393abe7be77dbd6c8d5/src/azure-cli/azure/cli/command_modules/appconfig/tests/latest/test_appconfig_commands.py#L1095

shouldn't the test case delete this file when test finishes?

Reply:
Good advice. I'll fix this.

Reply:
Yes. Removing files does not affect test.